p2panda-net-next/Cargo.toml

```diff
@@ -4,16 +4,18 @@ version = "0.1.0"
 edition = "2024"
 
 [dependencies]
-anyhow = "1.0.97"
 futures-lite = "2.6.1"
 iroh = { version = "0.93.2", default-features = false }
 iroh-base = "0.93.2"
 iroh-gossip = "0.93.1"
+n0-future = "0.3.0"
 p2panda-core = { path = "../p2panda-core", version = "0.4.0" }
-ractor = { version = "0.15.8", features = ["cluster"] }
```

Comment:
The `cluster` feature flag is only required if we have "remote" actors (on other machines). This and the connected `Message` trait can be removed.